### DISCUSSION TITLE
how to have agents and workflow "ask human input" with option "ALWAYS" and "TERMINATE" besides "NEVER"

### DISCUSSION THREAD

**Discussion by kevinisback** [microsoft/autogen#1377] [Q&A]
Hi, guys, I am doing the  autogenstudio to set up some agents that needs to ask further question from user after user pass the tasks. the problem I had right now is that there is only "NEVER" option in "ask human input" when setting agents. As result, the agent never ask me for further input. ( My prime goal is to have agents to ask me when it decides it needs to collect more information from me to proceed.) I am running on autogenstudio 0.0.33a0 with this problem.
I checked and found 0.0.25a0 version does have "ALWAYS" and "TERMINATE" but it has problem to create workflow. ( Workflow errors after creating. and no default workflow there). I played this on MacBook OS Monterey 12.7.1. Python environment is 3.11.7 Any insight or reply is highly appreciated! Cheers

**Comment by kevinisback** [comment]
![Snip20240123_4](https://github.com/microsoft/autogen/assets/36983509/2b3c016e-1736-49d2-bf71-dd4e902e7a96)
![Snip20240123_5](https://github.com/microsoft/autogen/assets/36983509/108ea3c0-e82a-4de2-ab1c-86949aefeed7)

**Comment by tonyppe** [comment]
Usually with software like this, not v1 yet, continuous iteration - you need the latest. 

That being said, I also have the same problem with no other options aside from NEVER. I made another post but as yet, no response to it. 

Some additional context would be helpful about this such as if NEVER is the only option designed going forward due to whatever reason - just so we know how to proceed because as it stands the documentation describes the options that are missing in recent versions so seems like a blocking bug but might not be.
